metadata.tvdb.com.python/resources/lib/cassettes/test_get_imdb_rating_and_votes.yaml

```diff
@@ -0,0 +1,603 @@
+interactions:
```

Comment:
should this be here?

Reply:
It could probably be omitted, as it's test related. Might be hard to do that reliable via .gitattributes

Reply:
Ah, I can probably ignore the folder

Reply:
Should be gone